hack/make.sh

```diff
@@ -88,7 +88,7 @@ if [ -z "$DOCKER_DEBUG" ]; then
 	LDFLAGS='-w'
 fi
 
-BUILDFLAGS=(${BUILDFLAGS} -tags "netgo osusergo static_build $DOCKER_BUILDTAGS")
+BUILDFLAGS=(${BUILDFLAGS} -tags "netgo osusergo static_build nri_no_wasm $DOCKER_BUILDTAGS")
```

Comment:
Should we add it to the other builds as well? We don't want it included in the non-static builds as well, right?

Reply:
ah! thanks!